ivy/array/creation.py

```diff
@@ -79,6 +79,31 @@ def full_like(
             an array having the same shape as ``self`` and where every element is equal
             to ``fill_value``.
 
+        Instance Method Examples:
+        -------------------
+
+        With int datatype:
+            >>> x = ivy.array([1,2,3])
```

Comment:
just to keep them formatting consistent, would you mind making a small change so that there's a blank line between subtitle and example code, then example code aligned with title, eg.

```
With int datatype:

>>> x = ivy.array([1,2,3])
...
```

Reply:
Please follow the convention for all examples, thanks!